mon-pix/tests/integration/components/steps_test.js

```diff
@@ -91,6 +91,26 @@ module('Integration | Component | steps', function (hooks) {
     });
 
     module('on the last page', function () {
+      test('should display information', async function (assert) {
+        // given
+        // when
+        const screen = await render(hbs`<CertificationInstructions::Steps/>`);
+        for (let i = 0; i < 4; i++) {
+          await click(screen.getByRole('button', { name: "Continuer vers l'écran suivant" }));
+        }
```

Comment:
Pour rendre plus comprehensible on pourrait encapsuler dans une methode (qui serait utilisée dans les 2 tests)
```suggestion
async getToLastScreen(){
        for (let i = 0; i < 4; i++) {
          await click(screen.getByRole('button', { name: "Continuer vers l'écran suivant" }));
        }
}
```